CMakeLists.txt

```diff
@@ -208,6 +208,20 @@ FetchContent_Populate(${short_extension_name}
   )
 list(APPEND Slicer_EXTENSION_SOURCE_DIRS ${${extension_name}_SOURCE_DIR})
 
+#-----------------------------------------------------------------------------
+# Srep2
+set(extension_name "SlicerSRep2")
```

Comment:
It shouldn't be an issue to have the extension called `SlicerSRep` instead of `SlicerSRep2`.

To accommodate the fact `SlicerSkeletalRepresentation` is already integrated, you could only have the `short_extension_name` set to `SRep2`.

Reply:
The name of the repository right now is SlicerSRep2. I would like the extension name to match the repo for clarity, and to be very clear that this is the successor to the other SRep extension still in SlicerSALT. SlicerSkeletalRepresentation vs SlicerSRep does not imply superseding, while I would argue that having the "2" in the name does.